src/node/fee_model.rs

```diff
@@ -0,0 +1,77 @@
+use zksync_basic_types::U256;
+use zksync_types::fee_model::{
+    BatchFeeInput, FeeModelConfigV2, PubdataIndependentBatchFeeModelInput,
+};
+use zksync_types::L1_GAS_PER_PUBDATA_BYTE;
+use zksync_utils::ceil_div_u256;
+
+const CONFIG: FeeModelConfigV2 = FeeModelConfigV2 {
+    minimal_l2_gas_price: 100000000,
+    compute_overhead_part: 0.0,
+    pubdata_overhead_part: 1.0,
+    batch_overhead_l1_gas: 800000,
+    max_gas_per_batch: 200000000,
+    max_pubdata_per_batch: 100000,
+};
+
+// TODO: consider using old fee model for the olds blocks, when forking
```

Comment:
Just in case, the old blocks will panic with the v2 batch fee input

Reply:
it's serious issue then

Reply:
Thanks, will test and fix

Reply:
Hmm, maybe I misunderstood you, what do you mean by that?

Reply:
I mean if you provide `PubdataIndependent` fee model into an old version of vm it will panic (i.e. with `vm_latest` it works, but it `vm_boojum_integration` it wont).

Reply:
I see, that's acceptable I think, because we are always using the latest VM currently